onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/OperatorRegistration.cpp

```diff
@@ -157,6 +162,48 @@ ONNX_OPERATOR_KERNEL_EX(
         // to make the re-use call.
         .Alias(0, 0),
     OptionalGetElement);
+
+ONNX_OPERATOR_KERNEL_EX(If,
+                        kOnnxDomain,
+                        7,
+                        kDmlExecutionProvider,
+                        (*KernelDefBuilder::Create())
+                            .InputMemoryType(OrtMemTypeCPUInput, 0)  // 'cond' needs to be on CPU
+                            .TypeConstraint("B", DataTypeImpl::GetTensorType<bool>())
+                            .TypeConstraint("V", DataTypeImpl::AllFixedSizeTensorTypes()),
+                        If);
```

Comment:
```suggestion
ONNX_OPERATOR_KERNEL_EX(
    If,
    kOnnxDomain,
    7,
    kDmlExecutionProvider,
    (*KernelDefBuilder::Create())
        .InputMemoryType(OrtMemTypeCPUInput, 0)  // 'cond' needs to be on CPU
        .TypeConstraint("B", DataTypeImpl::GetTensorType<bool>())
        .TypeConstraint("V", DataTypeImpl::AllFixedSizeTensorTypes()),
    If);
```

Weirdly inconsistent indentation.